pages/1.12/monitoring/logging/configure-task-logs/index.md

```diff
@@ -70,6 +70,12 @@ value (for example, 2^64 TB) will lead to undetermined results.</p>
 
 ## Logrotate Options
 
+<p class="message--important"><strong>IMPORTANT: </strong>
+As of DC/OS 1.12.4, these custom options are deprecated to prevent abuse
```

Comment:
s/deprecated/disallowed/ ?

Reply:
I think deprecated is more appropriate, since the plan is to add different flags to allow specific option.  Supplying this custom flag won't cause errors (if you specify an unknown `CONTAINER_LOGGER_*` flag, the task launch will fail), but it also will not be respected anymore.

Reply:
I always associated "deprecate" with something that still works but won't work in the near future. Maybe that's not how everyone uses that term.

Anyway, I think if the option will be ignored I think we should make that explicit. Maybe there are people like me who it still works.

Reply:
See: https://stackoverflow.com/questions/8111774/deprecated-meaning

Reply:
I see.  Perhaps "disabled" is better.